hud-data/weirdcorehud.json

```diff
@@ -6,7 +6,14 @@
   ],
   "releaseDate": "2024-04-06",
   "resources": [ "sfondo-hud" ,
```

Comment:
This name does not match up with the image's name (`sfondo-hud` vs `sfondo hud`). One or the other should be changed